File: Programmers/py/lev0/피자 나눠먹기(2)/solution.py
def solution(n):
    cnt = 1
    while (6 * answer % n):
        cnt += 1
    return cnt

##################################
# math 내장함수도 있음 
# gcd:최대공약수 , lcm :최소공배수
##################################
# import math
# def solution(n):
#     print(math.gcd(n, 6))
#     return (n * 6) // (math.gcd(n, 6)* 6)

File: Programmers/java/lev0/피자 나눠먹기(2)/solution.java
class Solution {
    final static int PIECE_NUM = 6;
    public int solution(int n) {
        int cnt =0, i =1;
        
        while (( PIECE_NUM * i) %n != 0 )
        {
            i++;
            cnt++;
        }
        return cnt+1;
    }
}

